source/core/NumericString.cpp

```diff
@@ -397,18 +347,18 @@ void TruncateLeadingZerosFromTimeString(StringRef buffer)
     }
 }
 
-void CreateMismatchedFormatSpecifierError(SubString* format, Int32 count, StaticTypeAndData* arguments, StringRef buffer, FormatOptions fOptions, Boolean* validFormatString, Boolean* parseFinished)
+void CreateMismatchedFormatSpecifierError(SubString* format, Int32 count, StaticTypeAndData* arguments, StringRef buffer, FormatOptions fOptions, Boolean* validFormatString, Boolean* parseFinished, ErrorCluster *errPtr)
```

Comment:
There are several callers, all of which would want to zero Buffer, so might as well let it be common here.